tutorials/receiving-money.md

```diff
@@ -0,0 +1,275 @@
+# Receiving Money
+
+## Typical Payment Request.
+
+Here is the typical API request to [initialize a transaction](https://developers.paystack.co/v2.0/reference#initialize-a-transaction).
+```
+curl https://api.paystack.co/transaction/initialize \
+-H "Authorization: Bearer SECRET_KEY" \
+-H "Content-Type: application/json" \
+-d '{"amount": 500000, "email": "customer@email.com"}' \
+-X POST
+```
+Transactions can be initialised via different payment methods - Paystack Inline, Paystack Standard.
+
+### Payment Parameters
+
+The most important fields in the request body are the `amount` to be charged and the customer `email`.
+
+If your application does not require customer emails, you can generate an `email` for the transaction with any unique identifier from your system like phone number or customer id: 08001234567@yoursite.com or j239dj3hn9393@yourapp.com.
+
+The identifier should be unique to the customer for all their transactions so that you can track the customer transaction history on your Paystack dashboard.
+
+You can add a `reference` to the request body which you use to identify this transaction on your own system. If you don't send a reference, Paystack will generate one for you for that transaction. Also transaction references must be unique for each transaction, otherwise you will get a "Duplicate transaction" error.
+
+The `amount` values that you pass to Paystack must be in kobo. To convert a Naira value to kobo, multiply by 100. So if you wanted to charge N100, the amount value to send will be 10000.
+
+###Adding Extra Information to the Payment Request
+
+Sometimes you want to pass extra information to your transaction. The initialization request body accepts a `metadata` object in which you can add other information you want. You can pass the object directly into the `metadata`. You can also use `custom_fields` array within the metadata. Custom fields is an array of objects. The values you pass in the `custom_fields` array will displayed in the transaction details on the Paystack Dashboard and on the customer's receipt.
```

Comment:
... array will **be** displayed in the transaction details 

grammatical error. Look at the word in bold